src/Auth0.AuthenticationApi/HttpClientAuthenticationConnection.cs

```diff
@@ -33,9 +36,12 @@ public class HttpClientAuthenticationConnection : IAuthenticationConnection, IDi
         /// </remarks>
         public HttpClientAuthenticationConnection(HttpClient httpClient = null)
         {
-            shouldDisposeHttpClient = httpClient == null;
+            ownHttpClient = httpClient == null;
```

Comment:
**Question:** You're referring to other class-level fields using `this.`, does it make sense to do the same here for consistancy? At first glance I thought this referred to a local field when I saw the other ones were prefixed.

Reply:
Oh, I've just seen it's because you have a local field for `httpClient` as well as a class-level field, so you need `this` in that case.

I guess it's fine, just took a moment to figure out.

Reply:
Yeah, some people like doing it for all but the default Visual Studio cleanup stuff only recommends it when you need to disambiguate.